Jint/DebuggerStatementHandling.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Jint
+{
+    public enum DebuggerStatementHandling
```

Comment:
Should this be in Runtime.Debugger namespace?

Reply:
Completely agree - I don't know why I only (briefly) considered the Runtime.Interpreter.Statements namespace. :-)